cognite/client/testing.py

```diff
@@ -113,6 +114,6 @@ def monkeypatch_cognite_client():
             >>>         assert "Something went wrong" == e.message
     """
     cognite_client_mock = CogniteClientMock()
-    CogniteClient.__new__ = lambda *args, **kwargs: cognite_client_mock
+    CogniteClient.__new__ = lambda *args, **kwargs: cognite_client_mock  # type: ignore[assignment]
```

Comment:
what does this mean?

Reply:
Mypy disallows assigning to methods (which is a good thing for the most part), but in this specific case we want to allow it.